pkg/geo/geogfn/segmentize_test.go

```diff
@@ -107,8 +107,8 @@ func TestSegmentize(t *testing.T) {
 			require.Equal(t, expectedGeog, modifiedGeog)
 		})
 	}
-	// Test for segment maximum length as negative for geometry collection.
-	t.Run(fmt.Sprintf("%s, maximum segment length: %v", segmentizeTestCases[9].wkt, 0.0), func(t *testing.T) {
+	// Test for segment maximum length as negative for multiline string.
+	t.Run(fmt.Sprintf("%s, maximum segment length: %f", segmentizeTestCases[9].wkt, 0.0), func(t *testing.T) {
```

Comment:
can we use mint a fresh `wkt`? i think referencing an array element can be deceptive.
it doesn't need to be in the run message, just

```
t.Run("errors when segmentize length <= 0", func ...
   ....
   _, err = Segementize(geog, 0)
}
```

is fine

Reply:
done!
Added fresh case for error run.
In both places.